src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rentCost = 40;
+
+  if (days >= 3 && days < 7) {
+    return rentCost * days - 20;
+  }
+
+  if (days >= 7) {
+    return rentCost * days - 50;
```

Comment:
Replace the hard-coded literal '7' with a constant to represent the minimum number of days for the long term discount. Also, replace '50' with a constant representing the long term discount amount.

Reply:
The same here.